draft-packages/tag/KaizenDraft/Tag/TagElmExtra.scss

```diff
@@ -0,0 +1,13 @@
+@import "./Tag.scss";
```

Comment:
I intentionally put this in a separate file, because this does not need to be shared with React. React should implement the tooltip on truncation (which is much easier in React) - issue here: https://github.com/cultureamp/kaizen-design-system/issues/1672.